provider/plugin.go

```diff
@@ -39,5 +39,6 @@ func (p *grpcPlugin) GRPCServer(broker *plugin.GRPCBroker, s *grpc.Server) error
 }
 
 func (p *grpcPlugin) GRPCClient(ctx context.Context, broker *plugin.GRPCBroker, c *grpc.ClientConn) (interface{}, error) {
+	//lintignore:R009
```

Comment:
@jrhouston so if I understand correctly, you wanted to keep this lintignore, right? Because this is the one place where a panic is reasonable to have in the provider? For the others, I'm going to try and resolve the issue by returning an error instead of panicking. (Let me know if I misunderstood your comment).

Reply:
Yup sounds good!